WSRequestHandler.cpp

```diff
@@ -1343,6 +1345,141 @@ void WSRequestHandler::HandleSetSceneItemCrop(WSRequestHandler* req) {
 }
 
 /**
+ * Gets the scene specific properties of the specified source item.
+ *
+ * @param {String (optional)} `scene-name` the name of the scene that the source item belongs to. Defaults to the current scene.
+ * @param {String} `item` The name of the source.
+ *
+ * @return TO POPULATE
```

Comment:
If nested properties how do I properly show the return types?

Reply:
It's a bit rough around the edges, but something like this will work.
Just put them in whatever order best makes sense as that'll be what's used.

```
@return {Object} `something`
@return {String} `something.expand` description
@return {String} `something.dong` description
```